SU-SWS/stanford_paragraph_types
# Earth 90

Adds Slide and Filmstrip paragraph types.

Steps to test:
1. Update stanford_components with branch EARTH-90
2. Update contrib module crop (drupal.org/project/crop) to version 8.x-1.2 (I had to do this manually before I could enable new features)
3. cd docroot/modules/stanford/stanford_components
4. git checkout --track origin/EARTH-90
5. drush cr all
6. Enable features Stanford Paragraph Slide and Stanford Paragraph Filmstrip
7. Add new paragraph field to a content type; see if you can use new paragraph types.
